core/src/main/scala/dx/core/ir/package.scala

```diff
@@ -146,6 +146,9 @@ object ExecutableKind {
   */
 trait RuntimeRequirement
 
+// TODO case class Application defined here
```

Comment:
Can this be removed?

Reply:
Fixed in [9c3ecae](https://github.com/dnanexus/dxCompiler/pull/291/commits/9c3ecae9726a388bdd2185c61bc605de4f5f251f)